src/Plugin/GraphQL/Fields/SearchAPISearch.php

```diff
@@ -85,6 +86,47 @@ private function addConditions($conditions) {
     }
   }
 
+  /**
+   * Adds a condition group to the Search API query.
+   *
+   * @condition_group
+   *  The conditions to be added.
+   */
+  private function addConditionGroup($condition_group_arg) {
+
+    // Set default conjunction.
+    $conjunction = "AND";
+
+    // Set conjunction from args.
+    if ($this->query->createConditionGroup($condition_group_arg['conjunction'])) {
+
```

Comment:
If i would have to nitpick some of these spacings right after if's and foreach I would remove them..not sure what the coding standards say about it though